### PR TITLE
chore(docs): Prepare sample for inclusion in cloud docs

### PR DESCRIPTION
**Pull request by morgante**
This prepares the sample for inclusion in the c.g.c. docs.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/simple/main.tf [hunk 1]
```diff
@@ -18,11 +18,17 @@ provider "google" {
   version = "~> 3.0"
 }
 
+# [START cloudrouter_create]
 module "cloud_router" {
-  source = "../../"
+  source  = "terraform-google-modules/cloud-router/google"
```
Comment (morgante) [inline examples/simple/main.tf:23]: Note: b/181274845

File: examples/simple/main.tf [hunk 2]
```diff
@@ -18,11 +18,17 @@ provider "google" {
...
+  version = "~> 0.4"
 
-  name    = "test-router"
+  name   = "test-router"
+  region = "us-central1"
+
+  # project = "my-project-id"
```
Comment (bharathkkb) [inline examples/simple/main.tf:29]: Would we swap `var.project` with "my-project-id" for docs?

**Review by bharathkkb — APPROVED** [review]

File: examples/simple/main.tf (hunk 2 above)
Comment (morgante) [reply examples/simple/main.tf:33]: No, this is just to provide a sample value.

**Pull request merged**